### PR TITLE
media-libs/liblastfm: Fix typo, EAPI=6

### PR DESCRIPTION
**Pull request by a17r**
@gentoo/kde

### CODE REVIEW

**Comment by johu** [comment]
What does the devmanual say about touching stable ebuilds?

**Comment by a17r** [comment]
I dialled back the EAPI. However, no change of installed files - the typo had no real-world effect.

File: media-libs/liblastfm/liblastfm-1.0.9.ebuild [hunk 1]
```diff
@@ -57,14 +57,14 @@ src_configure() {
...
-			$(cmake-utils_use_build test TESTS)
+			-DBUILD_FINGERPRINT=$(usex fingerprint)
+			-DBUILD_TESTS=$(usex test)
 		)
 		if [[ ${MULTIBUILD_VARIANT} = qt4 ]]; then
 			mycmakeargs+=(-DBUILD_WITH_QT4=ON)
 		fi
 		if [[ ${MULTIBUILD_VARIANT} = qt5 ]]; then
-			mycmakeargs+=(-DBUILD_WITH_OT4=OFF)
+			mycmakeargs+=(-DBUILD_WITH_QT4=OFF)
```
Comment (johu) [inline media-libs/liblastfm/liblastfm-1.0.9.ebuild]: Please only this change. The other changes are only cosmetic which should be done in a revision bump or version bump.

**Comment by johu** [comment]
Thanks picked as 72e7e7b271db71193d950692540c54c175e762dc

**Pull request closed without merging**